app.rb

```diff
@@ -0,0 +1,75 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative 'lib/database_connection'
+require_relative 'lib/peep_repository'
+require_relative 'lib/user_repository'
+
+DatabaseConnection.connect('chitter_test')
+
+class Application < Sinatra::Base
+ 
+  use Rack::Session::Pool
+
+
+  configure :development do
+    
+    register Sinatra::Reloader
+    also_reload 'lib/peep_repository'
+    also_reload 'lib/user_repository'
+  end
+
+  get '/' do 
+    repo_peep = PeepRepository.new
+    @peeps = repo_peep.all
+    erb :index
```

Comment:
just wondering if you need '()' i.e erb(:index)...? maybe just a thought but if it doesn't affect the functionality I guess it's alright.

Reply:
Great, didn't know you could refactor this.